hive/src/main/java/com/netease/arctic/hive/op/UpdateHiveFiles.java

```diff
@@ -98,6 +99,12 @@ public void commit() {
       this.partitionToCreate = getCreatePartition(this.partitionToDelete);
     }
 
+    if (PropertyUtil.propertyAsBoolean(
+        table.properties(),
+        HiveTableProperties.DELETE_UNTRACKED_HIVE_FILE,
+        HiveTableProperties.DELETE_UNTRACKED_HIVE_FILE_DEFAULT)) {
+      checkOrphanFilesAndDelete();
```

Comment:
When a user inserts data using native HiveSQL and wants to take advantage of Arctic's Hive Data sync feature, will this data be cleaned up?

Reply:
This clean action will only happen when spark insert overwrite be executed.